internal/tcpchecker/checkport.go

```diff
@@ -0,0 +1,78 @@
+package tcpchecker
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+)
+
+func CheckPort(address string, port int, reporter Reporter) (string, error) {
+	conn, err := net.DialTimeout("tcp", fmt.Sprintf("%s:%v", address, port), 1*time.Second)
+	if err != nil {
+		reporter.ReportPortCheck(Report{
+			Address: address,
+			Port:    port,
+			Open:    false,
+			Status:  "Connect error",
+			Error:   err,
+			Content: "",
+		})
+		return "", err
+	}
+
+	err = conn.SetDeadline(time.Now().Add(1 * time.Second))
+	if err != nil {
+		reporter.ReportPortCheck(Report{
+			Address: address,
+			Port:    port,
+			Open:    false,
+			Status:  "Set deadline error",
+			Error:   err,
+			Content: "",
+		})
+		return "", err
+	}
+
+	scanner := bufio.NewScanner(conn)
+
+	output := strings.Builder{}
+	for scanner.Scan() {
+		fmt.Fprintf(&output, "%s\n", scanner.Text())
+	}
```

Comment:
Actually the `strings.Builder` does not really give us an advantage over the string concatenation. It just makes it harder to understand IMO. :/ Maybe we should kist stock to that?